ethminer/MinerAux.h

```diff
@@ -525,8 +525,47 @@ class MinerCLI
 				BOOST_THROW_EXCEPTION(BadArgument());
 			}
 		}
+		else if ((arg == "--tstop") && i + 1 < argc)
+		{
+			try
+			{
+				m_tstop = stol(argv[++i]);
+			}
+			catch (...)
+			{
+				cerr << "Bad " << arg << " option: " << argv[i] << endl;
+				BOOST_THROW_EXCEPTION(BadArgument());
+			}
+		}
+		else if ((arg == "--tstart") && i + 1 < argc)
+		{
+			try
+			{
+				m_tstart = stol(argv[++i]);
+			}
+			catch (...)
+			{
+				cerr << "Bad " << arg << " option: " << argv[i] << endl;
+				BOOST_THROW_EXCEPTION(BadArgument());
+			}
+		}
 		else
 			return false;
+
+
+		// Sanity check --tstart/--tstop
+		if (m_tstop && m_tstop <= m_tstart)
```

Comment:
Implement a check for tstop to have a range from tstart of, at least, 10 degrees 
Otherwise a tstop of 41 and a tstart of 40 would be legit but will never start mining I assume

Reply:
Also do not let tstart be lower than 40.
A tstart of 10 would make no sense.

Reply:
* In your example - it will mine if the temperature equals 40 degrees.
* Forcing (tstart >= 40) and (tstart + 10 <= tstop):
Hmmm: I like it when software does not too much automatically.
Maybe we force the two values to be in a range of 30 to 100